content/docs/guides/browser-automation.md

```diff
@@ -15,9 +15,9 @@ Cucumber is not a browser automation tool, but it works well with the following
 
 WebDriver is designed to provide a simpler, more concise programming interface in addition to addressing some limitations in the Selenium-RC API. Selenium-WebDriver was developed to better support dynamic web pages where elements of a page may change without the page itself being reloaded. WebDriver's goal is to supply a well-designed object-oriented API that provides improved support for modern advanced web-app testing problems.
 
-[Selenium-WebDriver](https://docs.seleniumhq.org/docs/03_webdriver.jsp#setting-up-webdriver-project) can be used in multiple programming languages, including Java, JavaScript, Ruby and Kotlin.
+[Selenium-WebDriver](https://www.selenium.dev/documentation/en/selenium_installation/installing_selenium_libraries/#_java_) can be used in multiple programming languages, including Java, JavaScript, Ruby and Kotlin.
```

Comment:
Lets remove the auto-move from the end here.

Reply:
Hi , didn't get your point, if related to the new link, that's from new target URL with anchor. Otherwise, they're all from exsiting format.

Reply:
Yes, we don't want to auto-move people. Just hit the default area.

Reply:
This I think you missed, we don't need an auto-move, just remove the `#ref`

Reply:
Well, thanks for your patience, just updated the link without anchors.